cmd/podman/images/scan.go

```diff
@@ -0,0 +1,156 @@
+package images
+
+import (
+	"context"
+	"fmt"
+	"github.com/containers/podman/v2/cmd/podman/common"
+	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/containers/podman/v2/pkg/specgen"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"os"
+	"strings"
+)
+
+var (
+	scanDescription = `Scan a container image for known vulnerabilities.  The image name or digest can be used.`
+	scanCommand     = &cobra.Command{
```

Comment:
I don't think we should be adding any new primary commands.

This should be 

podman image scan ...
But no podman scan.